.eslintrc.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
-  extends: '@mate-academy/eslint-config',
+  extends: "@mate-academy/eslint-config",
   env: {
-    jest: true
+    jest: true,
   },
   rules: {
-    'no-proto': 0
+    "no-proto": 0,
+    quotes: "off",
```

Comment:
The `quotes` rule is set to "off". Ensure that this is intentional and aligns with the coding standards expected by the `@mate-academy/eslint-config`. If not, consider setting it to a specific value like "single" or "double" to enforce consistent quote usage.